DigitalLearningSolutions.Data.Tests/DataServices/ProgressDataServiceTests.cs

```diff
@@ -246,5 +249,37 @@ public void ClearAspProgressVerificationRequest_updates_aspProgress_records()
                 transaction.Dispose();
             }
         }
+
+        [Test]
+        public void Set_completion_date_should_update_db()
+        {
+            // Given
+            const int progressId = 1;
+            const int candidateIdForProgressRecord = 1;
+            const int customisationIdForProgressRecord = 100;
+            var completionDate = new DateTime(2020, 12, 25);
+
+            using var transaction = new TransactionScope();
+            try

```

Comment:
A pattern is tests has emerged with using these `try{} finally` statements along side `TransactionScope`.
With the use of the transaction these `try{} finally{}` statements can be removed.
We also should not actually need the `transaction.Dispose();` as the transactions should automatically be disposed at the end.